apps/remix-ide/src/app/tabs/locales/zh/panel.json

```diff
@@ -6,5 +6,7 @@
   "panel.maintainedByRemix": "由 Remix 维护",
   "panel.pluginInfo": "插件信息",
   "panel.linkToDoc": "文档链接",
-  "panel.makeAnissue": "提交 issue"
+  "panel.makeAnissue": "提交 issue",
```

Comment:
You don't need to add those in all language files. It will take English by default anyway. Just add the label "needs translation".